monitoring/grafana/dashboards/hosts-overview.json

```diff
@@ -843,8 +845,8 @@
       "30d"
     ]
   },
-  "timezone": "",
-  "title": "Host Overview",
-  "uid": "y0KGL0iZz",
-  "version": 3
+  "timezone" : "",
+  "title" : "Host Overview",
+  "uid" : "y0KGL0iZz",
+  "version" : 3
```

Comment:
This seems to fully rewrite the json file with some whitespace changes. Please remove this edit.